### PR TITLE
Update control systems

### PR DESCRIPTION
**Pull request by skrueger-ftc**
Started as wanting to add a warning about the upcoming illegalization of Androids phones as RC controllers, ([See note in RE06](https://firstinspiresst01.blob.core.windows.net/first-in-show-ftc/game-manual-part-1-traditional.pdf)). I ended up updating most of the page since it seemed some of the information on the page hasn't been updated past ~Skystone.

### CODE REVIEW

File: source/docs/power-and-electronics/control-system.rst [hunk 1]
```diff
@@ -3,47 +3,52 @@
 Control Systems
 ===============
 
-The FTC control system is based on using Android phones as a ":term:`Robot Controller`" and a ":term:`Driver Station`." The ":term:`Robot Controller`" phone is mounted on the robot and connected by a USB cable to a special "Hub", which in turn is connected to motors, servos, and sensors.
+The FTC control system is based around a ":term:`Robot Controller`" and a ":term:`Driver Station`". The :term:`Robot Controller` is mounted on the robot and are connected to special "Hub(s)", which in turn connect to motors, servos, and sensors. The :term:`Robot Controller` is linked to the :term:`Driver Station` through WiFi Direct.
```
Comment (NoahAndrews) [inline source/docs/power-and-electronics/control-system.rst:6]:
```suggestion
The FTC control system is based around a ":term:`Robot Controller`" and a ":term:`Driver Station`". The :term:`Robot Controller` is mounted on the robot and is connected to special "Hub(s)", which in turn connect to motors, servos, and sensors. The :term:`Robot Controller` is linked to the :term:`Driver Station` through WiFi Direct.
```

Comment (NoahAndrews) [reply source/docs/power-and-electronics/control-system.rst:6]: This paragraph doesn't really account for the fact that the Robot Controller can itself be a REV Hub. Additionally, the Control Hub does not use Wi-Fi Direct in any way. Instead, it broadcasts a standard Wi-Fi access point.

File: source/docs/power-and-electronics/control-system.rst [hunk 2]
```diff
@@ -3,47 +3,52 @@
...
 
-The two major manufacturers of control hubs for FTC are Modern Robotics and REV; however, nearly all teams are shifting over to REV because it is cheaper, simpler, and more reliable.
+REV Robotics is the sole manufacturer of legal FTC control systems. The REV Expansion Hub connects to motors, servos, sensors, and a :term:`Robot Controller`. A REV Control Hub has the same functionality of an Expansion Hub but with a built-in :term:`Robot Controller`.
```
Comment (NoahAndrews) [inline source/docs/power-and-electronics/control-system.rst:8]:
```suggestion
REV Robotics is the sole manufacturer of legal FTC control system components. The REV Expansion Hub connects to motors, servos, sensors, and a :term:`Robot Controller`. A REV Control Hub has the same functionality of an Expansion Hub but with a built-in :term:`Robot Controller`.
```

**Review by NoahAndrews — CHANGES_REQUESTED** [review]

File: source/docs/power-and-electronics/control-system.rst (hunk 1 above)
Comment (skrueger-ftc) [reply source/docs/power-and-electronics/control-system.rst:6]: Thoughts:
The FTC control system is based around a ":term:`Robot Controller`" and a ":term:`Driver Station`". The :term:`Robot Controller` is mounted on the robot and handles motors, servos, sensors, and input from the drivers' controllers. The :term:`Driver Station` remain with the drive team and connects to the :term:`Robot Controller` over WiFi.

Adds some ambiguity to the Robot Controller definition, just giving a very basic overview of its purpose. This leaves the second paragraph for actually describing the hubs. Also fixes up the WiFi Direct mistake.

Comment (abidingabi) [inline source/docs/power-and-electronics/control-system.rst:6]:
```suggestion
The FTC control system is based around a ":term:`Robot Controller`" and a ":term:`Driver Station`". The :term:`Robot Controller` is mounted on the robot. It is either embedded within or connected to special "Hub(s)", which in turn connect to motors, servos, and sensors. The :term:`Robot Controller` is linked to the :term:`Driver Station` through WiFi Direct.
```

File: source/docs/power-and-electronics/control-system.rst [hunk 3]
```diff
@@ -3,47 +3,52 @@
...
 
-This is the standard control system for teams starting out in FTC. The REV Expansion Hub is reliable, as long as proper strain relief and wiring is carried out. This includes the :term:`USB Retention Mount`, as well as 3D printing :term:`XT30` stress relief mounts.
+This is the standard control system for teams starting out in FTC.
 
-The Expansion Hub connects to the :term:`Robot Controller` phone through the mini USB port, and the :term:`RC <Robot Controller>` phone is linked to the DS (:term:`Driver Station`) phone through WiFi Direct. For more information on setting up the Expansion Hub and configuring the robot, head to `REV Robotics' Technical Resources Expansion Hub Guide <https://docs.revrobotics.com/duo-control/legacy/expansion-hub-gs>`_.
+The Control Hub connects to the Expansion Hub through either a RS-485 port on both hubs or a USB-A port on the Control Hub to the mini USB port on the Expansion Hub. In either case, proper strain relief and cable management should be used.
```
Comment (abidingabi) [inline source/docs/power-and-electronics/control-system.rst:30]:
```suggestion
The Control Hub connects to the Expansion Hub through either a RS-485 connection, or a USB-A (Control Hub side) to mini-USB (Expansion Hub-side) connection. In either case, proper strain relief and cable management should be used.
```

File: source/docs/power-and-electronics/control-system.rst [hunk 4]
```diff
@@ -3,47 +3,52 @@
...
 
-REV Control Hub + REV Expansion Hub
------------------------------------
+RC Phone + REV Expansion Hub(s)
+-------------------------------
 
 .. warning:: It is of vital importance to update the firmware on REV Expansion hubs to at least version 1.8.2. It includes better protection against disconnects and improves program performance. See the `REV Expansion Hub firmware update docs`_.
 
-Note: The Control Hub will be legal for all teams from the 2020-2021 FTC season onwards. In 2019-2020, the Control Hub was not legal for all teams as it was undergoing a pilot program.
+The Expansion Hub connects to the :term:`Robot Controller` phone through the mini USB port. The REV Expansion Hub is reliable, as long as proper strain relief and wiring is carried out. This includes the :term:`USB Retention Mount`, as well as 3D printing :term:`XT30` stress relief mounts.
```
Comment (abidingabi) [inline source/docs/power-and-electronics/control-system.rst:43]:
```suggestion
The Expansion Hub connects to the :term:`Robot Controller` phone through its mini USB port. The REV Expansion Hub is reliable, as long as proper strain relief and wiring is carried out. This includes the :term:`USB Retention Mount`, as well as 3D printing :term:`XT30` stress relief mounts.
```

File: source/docs/power-and-electronics/control-system.rst [hunk 5]
```diff
@@ -3,47 +3,52 @@
 Control Systems
 ===============
 
-The FTC control system is based on using Android phones as a ":term:`Robot Controller`" and a ":term:`Driver Station`." The ":term:`Robot Controller`" phone is mounted on the robot and connected by a USB cable to a special "Hub", which in turn is connected to motors, servos, and sensors.
+The FTC control system is based around a ":term:`Robot Controller`" and a ":term:`Driver Station`". The :term:`Robot Controller` is mounted on the robot. It is either embedded within or connected to special "Hub(s)", which in turn connect to motors, servos, and sensors. The :term:`Robot Controller` is linked to the :term:`Driver Station` through WiFi Direct.
```
Comment (abidingabi) [inline source/docs/power-and-electronics/control-system.rst:6]:
```suggestion
The FTC control system is based around a ":term:`Robot Controller`" and a ":term:`Driver Station`". The :term:`Robot Controller` is mounted on the robot. It is either embedded within or connected to special "Hub(s)", which in turn connect to motors, servos, and sensors. The :term:`Robot Controller` is linked to the :term:`Driver Station` through WiFi or WiFi Direct.
```

File: source/docs/power-and-electronics/control-system.rst [hunk 6]
```diff
@@ -3,47 +3,52 @@
...
 
-The two major manufacturers of control hubs for FTC are Modern Robotics and REV; however, nearly all teams are shifting over to REV because it is cheaper, simpler, and more reliable.
+REV Robotics is the sole manufacturer of legal FTC control system components. The REV Expansion Hub connects to motors, servos, sensors, and a :term:`Robot Controller`. A REV Control Hub has the same functionality of an Expansion Hub but with a built-in :term:`Robot Controller`.
 
-More information about the FTC Control system can be found below:
+More information about the FTC Control System can be found below:
 
-- `Official control system Wiki on GitHub <https://github.com/FIRST-Tech-Challenge/FtcRobotController/wiki>`_
+- `Official Control System docs on FTC Docs <https://ftc-docs.firstinspires.org/en/latest/programming_resources/shared/control_system_intro/The-FTC-Control-System.html>`_
```
Comment (abidingabi) [inline source/docs/power-and-electronics/control-system.rst:12]:
```suggestion
- `Official Control System Documentation on FTC Docs <https://ftc-docs.firstinspires.org/en/latest/programming_resources/shared/control_system_intro/The-FTC-Control-System.html>`_
```

**Pull request merged**